doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -8,306 +8,309 @@
+| [7.11.7](../../../ref_model/chapters/chapter07.md#7117-monitoring-and-security-audit) | sec.mon.008 | The Platform must Monitor and Audit externally exposed interfaces for illegal access (attacks) and take corrective security hardening measures |  | |
+| [7.11.7](../../../ref_model/chapters/chapter07.md#7117-monitoring-and-security-audit) | sec.mon.009 | The Platform must Monitor and Audit service handling for various attacks (malformed messages, signalling flooding and replaying, etc.) and take corrective actions accordingly |  | |
+| [7.11.7](../../../ref_model/chapters/chapter07.md#7117-monitoring-and-security-audit) | sec.mon.010 | The Platform must Monitor and Audit running processes to detect unexpected or unauthorized processes and take corrective actions accordingly |  | |
+| [7.11.7](../../../ref_model/chapters/chapter07.md#7117-monitoring-and-security-audit) | sec.mon.011 | The Platform must Monitor and Audit logs from infrastructure elements and workloads to detected anomalies in the system components and take corrective actions accordingly |  |
+| [7.11.7](../../../ref_model/chapters/chapter07.md#7117-monitoring-and-security-audit) | sec.mon.012 | The Platform must Monitor and Audit Traffic patterns and volumes to prevent malware download attempts |  | |
+| [7.11.7](../../../ref_model/chapters/chapter07.md#7117-monitoring-and-security-audit) | sec.mon.013 | The monitoring system must not affect the security (integrity and confidentiality) of the infrastructure, workloads, or the user data (through back door entries). |  |
+| [7.11.7](../../../ref_model/chapters/chapter07.md#7117-monitoring-and-security-audit) | sec.mon.014 | The Monitoring systems should not impact IAAS, PAAS, and SAAS SLAs including availability SLAs |  |
+| [7.11.7](../../../ref_model/chapters/chapter07.md#7117-monitoring-and-security-audit) | sec.mon.015 | The Platform must ensure that the Monitoring systems are never starved of resources |  |
+| [7.11.7](../../../ref_model/chapters/chapter07.md#7117-monitoring-and-security-audit) | sec.mon.016 | The Platform Monitoring components should follow security best practices for auditing, including secure logging and tracing |  |
+| [7.11.7](../../../ref_model/chapters/chapter07.md#7117-monitoring-and-security-audit) | sec.lcm.017 | The Platform must Audit systems for any missing security patches and take appropriate actions |  |
+| [7.11.8](../../../ref_model/chapters/chapter07.md#7118-compliance-with-standards) | sec.std.001 | The Cloud Operator **should** comply with Center for Internet Security CIS Controls ([https://www.cisecurity.org/](https://www.cisecurity.org/)) | |
+| [7.11.8](../../../ref_model/chapters/chapter07.md#7118-compliance-with-standards) | | [Q: Are we going to verify compliance w Controls? If not, then why a “must” – but making it a “should” implies only guidance and not a control.] |  |
```

Comment:
I think this is not a requirement.

Reply:
Good spot - removed in https://github.com/cntt-n/CNTT/pull/1524/commits/20414d04b5fd753639eac54223ddb12649947d7e.